### DISCUSSION TITLE
"Confirm merge" button doesn't work

### DISCUSSION THREAD

**Discussion by pasere-smeup** [community/community#59314] [Pull Requests] [answered]
### Select Topic Area

Bug

### Body

Hi,

This morning the "Confirm merge" button doesn't work anymore.
Please help me...

**Comment by GilEliyahu23** [comment]
Same case also for me

**Reply by GilEliyahu23** [reply]
Now it works

**Comment by pasere-smeup** [comment]
Now it worked!
I don't know why.

**Comment by willsmythe** [comment, marked as answer]
There was a brief continuation of the incident yesterday ([link](https://www.githubstatus.com/incidents/tcp7119jvq7y)) that caused this. The root cause has been identified and fixed. Sorry for the inconvenience here!

**Reply by pasere-smeup** [reply]
Thank you very much!
Have a good day.
